mathesar/api/db/viewsets/tables.py

```diff
@@ -224,3 +224,15 @@ def existing_import(self, request, pk=None):
         )
         table_data = serializer.data
         return Response(table_data)
+
+    @action(methods=['post'], detail=True)
+    def mapper(self, request, pk=None):
```

Comment:
This is not a name that will be understandable without massive context. I suggest: `map_imported_columns` or the like.